src/main/java/uk/gov/justice/probation/courtcasematcher/service/CourtCaseService.java

```diff
@@ -49,7 +49,11 @@ public void createCase(CourtCase courtCase, SearchResult searchResult) {
     }
 
     public void saveCourtCase(CourtCase courtCase) {
-        restClient.putCourtCase(courtCase.getCourtCode(), courtCase.getCaseNo(), courtCase);
+        try {
+            restClient.putCourtCase(courtCase.getCourtCode(), courtCase.getCaseNo(), courtCase).block();
```

Comment:
In general, we tend to avoid comments in code these days but I wonder about whether it's worth a little note about why we are doing block() here because that's not normal and running against the async nature of reactive. Just a thought. Happy if you ignore it.